src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/CSharpFormattingOptions2.cs

```diff
@@ -290,11 +290,7 @@ private static Option2<T> CreateOption<T>(OptionGroup group, string name, T defa
     internal static readonly ImmutableArray<IOption2> UndocumentedOptions = [CollectionExpressionWrappingLength];
 }
 
-#if CODE_STYLE
-internal enum LabelPositionOptions
-#else
-public enum LabelPositionOptions
-#endif
+internal enum LabelPositionOptionsInternal
```

Comment:
these are now internal *and* renamed (so no conflicts anymore).  all shared code now uses the internal option.